.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Generate lockfile if needed
+        run: |
+          if [ ! -f pnpm-lock.yaml ]; then
+            echo "Generating pnpm-lock.yaml"
+            pnpm install --no-frozen-lockfile
+          fi
+
+      - name: Setup Turborepo cache
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)